lib/models/NotificationConfiguration.js

```diff
@@ -254,7 +254,7 @@ class NotificationConfiguration {
      */
     static getConfigXML(config) {
         const xmlArray = [];
-        if (config) {
+        if (config && Object.keys(config).length > 0) {
```

Comment:
Why not checking instead `if (config && config.queueConfig)`?

Reply:
cos I copied the logic over from cloudserver where it made more sense. good catch